docs/insights/index.md

```diff
@@ -0,0 +1,88 @@
+
+DORA recommends an approach to measuring software delivery that relies on five metrics:
+
+*Tempo*
+- Lead time for changes (LT)
+- Deployment frequency (DF)
+
+*Stability*
+- Change failure rate (CFR)
+- Mean time to recovery (MTTR)
+
+*Operation*
+- Reliability
+
+Throughput metrics measure the health of your deployment pipeline, while the stability indicators help you understand the quality of your software and delivery pipeline. In addition to the four classic DORA metrics that measure software delivery performance, DORA added a new measure in 2021 for operational performance.
+
+## Octopus built-in DORA metrics with DevOps Insights
+
+Release 2022.3 adds out-of-the-box support for the following DevOps metrics:
+
+**Deployment Lead Time**
+
+The time between the creation date of the release immediately following the previously successful release and the completion date of the deployment.
+
+**Deployment Failure Rate**
+
+The percentage of deployments that fail to deploy, require guided failure or have their release marked as "Do not promote"
+
+**Deployment Frequency**
+
+How many deployments occur to the target environment.
+
+**Mean Time to Recovery**
+
+How long it takes to recover a failed deployment with a subsequent successful deployment.
+
+> Some of these metrics differ slightly from the textbook DORA metrics given the data available.
+
+Together these metrics help you qualify the results of your DevOps performance, as well as gain insights into areas for future improvement.
+
+- Get better visibility into the performance of your projects and teams
+- Eliminate “gut feel” and enable data-informed decisions to drive improvement and determine if a new process is working
+- Review and collect data over time to highlight the path to delivering greater value, faster
+- Help introduce change with data and collaboration to make a business case
+- Share successes and learn from failures to continuously improve
+
+## Understand performance of your projects with Project level Insights
+
+![Project Insights Deployment Frequency](images/project.png)
+
+Project level insights are available as a new tab in every project so you can understand performance of your projects across Channels, Environments and Tenants. Each metric can be seen at a summary level and insights can also be filtered to time frames including last month, quarter and year, channels and environments, as well as being exported into CSV.
+
+Project level insights are available to all customers out-of-the-box meaning you don’t have to buy or subscribe to another tool. If you’re already a user, Octopus already has all the data it needs to help you uncover rich insights based on your deployment history.
```

Comment:
Grammarly suggestions and a suggestion to move the image to after the paragraphs

```suggestion
Project level insights are available as a new tab in every project so you can understand the performance of your projects across Channels, Environments, and Tenants. Each metric can be seen at a summary level, and insights can also be filtered to time frames, including last month, quarter, year, channels, and environments, as well as being exported into CSV.

Project level insights are available to all customers out-of-the-box, meaning you don’t have to buy or subscribe to another tool. If you’re already a user, Octopus has all the data it needs to help you uncover rich insights based on your deployment history.

![Project Insights Deployment Frequency](images/project.png)
```